civictechprojects/static/css/_vars.scss

```diff
@@ -20,36 +20,104 @@ $color-disabled-border: #a8a8a8; // for e.g. a disabled state button border
+$color-brand-80: #f06d23;
+
+//background
+$color-neutral-10: #ffffff;
+$color-neutral-15: #f6f7f8;
+$color-neutral-20: #f5f6f6;
+$color-neutral-30: #eaeef0;
+$color-neutral-45: #8c8c8c; // rgba(25,25,25,0.5) to rgb; may not be correct, waiting for design response 04/2022
+
+// border
+$color-neutral-35: #d4d4d4;
+$color-neutral-50: #bdbdbd;
+$color-neutral-60: #828282;
+$color-neutral-75: #4e5154;
+
+// secondary colors | semantics //
+
+// info
+$color-blue-10: #e6effa;
+$color-blue-20: #ccdced; //rgba(2,79,166,0.2) to rgb
+$color-blue-40: #81b1e6;
+$color-blue-80: #0362cd;
+$color-blue-90: #024fa6;
+$color-blue-100: #02438c;
+$color-purple-100: #662d91;
+
+// success
+$color-green-10: #e6f7ef;
+$color-green-20: #cce6da; //rgba(1,128,71,0.2) to rgb
+$color-green-80: #018047;
+$color-green-100: #01562f;
+
+//caution-warning
+$color-orange-10: #fdf5da;
+$color-orange-20: #f6e1d2; //rgba(211, 104, 32, 0.2) to rgb
+$color-orange-80: #d36820; // see design system notes on when to use this
+$color-orange-100: #b0571b; // see design system notes on when to use this
+
+//critical-error
+$color-red-10: #fdf0f0;
+$color-red-20: #f9cece; // rgba(227,12,12,0.2) to rgb
+$color-red-50: #e30c0c;
+$color-red-80: #b30909;
+$color-red-100: #970808;
+
+//focus ring (css :focus outline)
+$color-blue-50: #4f91dc;
+
+// # end color scales
+// # begin semantic variable names (use these)
 
-//second, declare semantic names -- declare THESE in your CSS, don't declare color name vars directly.
 $color-feedback-background-info-weakest: $color-blue-10; // "Background color for informational messages"
 $color-feedback-background-success-weakest: $color-green-10; // "Background color for success messages and statuses"
 $color-icon-success-strong: $color-green-80; // "Default icon color for success icon
 $color-border-weak: $color-neutral-50; // "Border color for small UI components"
 $color-text-strong: $color-neutral-90; // "Secondary text color for headers and subtitles"
 $color-text-strongest: $color-neutral-100; // "Primary text color for H1/H2. Text color for most UI Components."
-// end temporary holding zone 
+$color-background-default: $color-neutral-15; //site body
+$color-background-light: $color-neutral-10; //individual components
+
+
+// # end semantic variable names
+
+// #####
+// End Design System 
+// #####
+
 
+//note: everything below here should be checked and brought into DS, these are live values
 
 // ## link colors (design system "Secondary Colors - Blue Color Scheme - used for text links across dlab, implemented October 6, 2021)
-// note that these are also in _bootstrapoverride, overriding $link-color and $link-hover-color 
+// these are also in _bootstrapoverride, overriding $link-color and $link-hover-color 
 $color-link: #0362cd; // primary link color, info color
 $color-link-hover: #024fa6; // use with underline
 $color-link-active: #02438c; // when something in a pressed state, also use underline
 $color-link-visited: #662d91; 
```

Comment:
Just wonder, if we need any states for the visited links? :D

Reply:
I don't think we do, but if you find a link where the visited state isn't doing what you expect, let me know and I'll investigate it.